galley/pkg/config/analysis/diag/messages.go

```diff
@@ -14,7 +14,32 @@
 
 package diag
 
-import "sort"
+import (
+	"encoding/json"
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/ghodss/yaml"
+)
+
+const (
+	LogFormat  = "log"
+	JSONFormat = "json"
+	YAMLFormat = "yaml"
+)
+
+var (
+	MsgOutputFormatKeys = []string{LogFormat, JSONFormat, YAMLFormat}
+	MsgOutputFormats    = make(map[string]bool)
+)
+
+func init() {
+	sort.Strings(MsgOutputFormatKeys)
```

Comment:
Why do we need to do this?

Reply:
It was on line 105 in the original [code](https://github.com/istio/istio/pull/25996/files/0ad7cc516a4fd09163cd48033a23be4407f140fb#diff-b3e374e4ac15f6818ab702c5172ce519L105). I kept it as it was but sure we can remove this.

Reply:
You can use the blame interface in the original file to see who added a particular line at what commit/pull. This was added in #24212 and is needed.